fix_simulation/Readme.md

```diff
@@ -0,0 +1,55 @@
+# Trade Simulation
+This project contains a simulated trading application that interacts with a FIX (Financial Information eXchange) server. It simulates placing, modifying, and canceling orders for two clients.
+
+## Prerequisites
+- Docker and Docker Compose installed on your system
+- Python 3.7 or higher
+
+## Setup
+1. Clone this repository to your local machine.
+
+2. Create a `.env` file in the root directory of the project with the following content:
+
+```
+TRUEX_USER1_MNEMONIC=your_user1_mnemonic
+TRUEX_USER1_KEY_ID=your_user1_key_id
+TRUEX_USER1_KEY_SECRET=your_user1_key_secret
+TRUEX_USER2_MNEMONIC=your_user2_mnemonic
+TRUEX_USER2_KEY_ID=your_user2_key_id
+TRUEX_USER2_KEY_SECRET=your_user2_key_secret
+```
+
+Replace the placeholder values with your actual credentials.
+
+3. Ensure that the `client1.cfg` and `client2.cfg` files are present and correctly configured.
+
+## Running the Trade Simulation
+To run the trade simulation using Docker Compose:
+
+1. Open a terminal and navigate to the project directory.
+
+2. Run the following command: `docker-compose up --build`
+
+
+## TRUEX Environments
+1. Development - 10.10.10.11
+2. User Acceptance Test(UAT) - 10.10.20.11
```

Comment:
This is a public repo and these are internal IPs. Clients likely won't have access to the development environment since it probably going to be too chaotic for them. Just uat.truex.co is all they should need for now.